src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Email/EmailConfigurationType.php

```diff
@@ -102,6 +104,27 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('log_emails', SwitchType::class, [
                 'label' => $this->trans('Log Emails', 'Admin.Advparameters.Feature'),
             ])
+            ->add('dkim_enable', SwitchType::class, [
+                'label' => $this->trans('Enable DKIM', 'Admin.Advparameters.Feature'),
```

Comment:
```suggestion
                'label' => $this->trans('DKIM', 'Admin.Advparameters.Feature'),
```

Reply:
Please note that this suggestion should only be applied if it is possible to change 'Yes' and 'No' to 'Enabled' and 'Disabled' in the toggle. If it is not possible for now, then we should keep 'Enable DKIM'. Thanks!

Reply:
Hi @Julievrz, it's possible to change it, but it looks weird in my language for example. WDYT? Ping @NeOMakinG

![width](https://user-images.githubusercontent.com/6097524/111767571-2e249f00-88a7-11eb-8c45-622e5c031408.JPG)

```
->add('dkim_enable', SwitchType::class, [
                'choices' => [
                    'Disabled' => false,
                    'Enabled' => true,
                ],
            ])
```

Reply:
Hi @Hlavtox, I thought you were using the new toggles. Don't you?

![image](https://user-images.githubusercontent.com/70583503/111784319-97161200-88bb-11eb-9db6-014cf2fee7bd.png)

Reply:
At least that's the toggles I see when testing your PR. :)